source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,192 @@
+.. _realm-sync-migrate-modes:
+
+=============================================
+Migrate Partition-Based Sync to Flexible Sync
+=============================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 3
+   :class: singlecol
+
+You can automatically migrate App Services Apps that use Partition-Based Sync
+to Flexible Sync. We recommend doing this, as Partition-Based Sync is an older
+mode.
+
+Other than upgrading the SDK version, migrating sync modes doesn't require any 
+changes to your client app code. Your App's backend will automatically translate
+the client's Partition-Based Sync requests into Flexible Sync.
+.. QUESTION: is there a performance hit for the translation?
+
+Migrating results in Partition-Based Sync clients communicating with a
+Flexible Sync App Services App. You cannot reverse this and have Flexible Sync
+clients communicate with a Partition-Based Syn App Services App.
+
+Requirements
+------------
+
+- Your App must currently use Partition-Based Sync
+- Linked Atlas cluster must be running mongodb 5.0+ for Flexible Sync compatibility
+- Linked Atlas cluster must have time-based oplog setting enabled
```

Comment:
this carries with it two other implicit requirements:
- linked cluster must have storage auto-scaling enabled
- linked cluster must be M10+ to access this setting

Reply:
Ooh, those are good details. I remember you mentioning auto-scaling, but I didn't know you need M10+ to access it.

I'll add these to make them clear to readers.